webapp/src/Twig/TwigExtension.php

```diff
@@ -174,6 +175,16 @@ public function printtimediff(float $start, ?float $end = null): string
         return Utils::printtimediff($start, $end);
     }
 
+    public function printremainingminutes(float $start, float $end): string
+    {
+        $minutesRemaining = round(($end - $start)/60, 0);
+        if ($minutesRemaining < 1) {
+            return 'less than 1 minute to go';
+        } else {
+            return $minutesRemaining . ' minutes to go';
```

Comment:
How does this ever display `1 minute to go` like in your screenshot?

Reply:
Urgs, that was because I can't do git properly.